configure.ac

```diff
@@ -1,41 +1,628 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.63)
-AC_INIT(ovis, 4.3.3, ovis-help@sandia.gov)
+AC_INIT(ovis-ldms, 4.3.3, ovis-help@sandia.gov)
```

Comment:
Do we want to rename the package in this PR?  This means that the dist tarball will be named "ovis-ldms-X.X.X.tar.gz" int the future. Are packages expected to rename as well?

Reply:
i would prefer to leave that for another time. i’m in the midst of cleaning up and adding spec files that handle the codes as software collections without double full configure.

Reply:
What I mean is that this PR _does_ change the package name, and perhaps that should have been left to some other time.  If this PR introduces a name change, then we probably need to discuss the implications now.  If that was accidental and the PR will be updated to not change the name, then we can postpone that discussion for the future.

Reply:
Ah, ok. yes, the merge of ovis-lib, ovis-ldms, and ovis all to ovis-ldms (in terms of autoconf package declarations) is to preserve all the existing third-party init scripting uses of the ldms pkglibdir location /usr/lib64/ovis-ldms where plugins are found. If we want to change that pkglibdir, I think that is something that should only occur in v5.

Reply:
I see. OK, that is probably fine then for v4.  It is probably too much trouble to introduce a "alternatepkglibdir" everywhere.  We'll just deal with the dist tarball being misnamed for now.

Reply:
One could argue that ovis-ldms is the correct name for our github (ovis-hpc) ldms product. Ovis is a project with a group of products (ldms, baler, sos, several others historically) and if anyone forks ldms, then those here remain ovis-ldms, ovis-sos, etc and someone may choose to release/package xyz-ldms. Obviously, we'd prefer everything developed is contributed to the community version instead of forks appearing.